### ISSUE TITLE
python binding for director

### ISSUE THREAD

**Issue by sunit1409** [RobotLocomotion/director#624]
Is there any python binding ?

**Comment by patmarion** [comment]
Yes.  Director is implemented in c++ and python.  Nearly all of the c++ code in Director and its dependencies VTK and Qt are wrapped for Python.  When you run the Director application you can press F8 or go to the Tools menu to open the python console to interact with objects in Python.
